web/observing_changes/observing_changes_to_light_dom_children/my_element.dart

```diff
@@ -0,0 +1,33 @@
+@HtmlImport('my_element.html')
+library my_element;
+
+import 'dart:html';
+import 'package:web_components/web_components.dart' show HtmlImport;
+import 'package:polymer/polymer.dart';
+
+@PolymerRegister('my-element')
+class MyElement extends PolymerElement {
+  int _counter = 1;
+
+  @property String message = '';
+
+  MyElement.created() : super.created();
+
+  void ready() {
+    new PolymerDom($['content']).observeNodes(_childrenUpdated);
+  }
+
+  void _childrenUpdated(PolymerDomMutation mutations) {
+    set(
```

Comment:
I'm pretty sure this is intentional, this looks especially weird because the method name is so short.
For example here dart_style#444 also the first parameter is already in the 2nd line.